src/Parameters.jl

```diff
@@ -0,0 +1,46 @@
+"""
+    Interface with free parameters needed by KiD, CloudMicrophysics and Thermodynamics
+
+    Overwriting the default values happens in the driver file.
+    Here we are just creating the KinematicParameters struct and
+    forwarding the parameters definitions to Thermodynamics and CloudMicrophysics definitions.
+"""
+module Parameters
+
+import Thermodynamics as TD
+import CloudMicrophysics as CM
+
+abstract type AbstractKinematicParameters end
+const AKP = AbstractKinematicParameters
+
+# Define KiD parameters
+Base.@kwdef struct KinematicParameters{FT, MP} <: AKP
+    w1::FT
+    t1::FT
+    microphys_params::MP
+end
+thermodynamics_params(ps::AKP) = CM.Parameters.thermodynamics_params(ps.microphys_params)
+microphysics_params(ps::AKP) = ps.microphys_params
+
+Base.eltype(::KinematicParameters{FT}) where {FT} = FT
+w1(ps::AKP) = ps.w1
+t1(ps::AKP) = ps.t1
+
+# Forward parameters to Thermodynamics
+const TDPS = TD.Parameters.ThermodynamicsParameters
+for var in fieldnames(TDPS)
+    @eval $var(ps::AKP) = TD.Parameters.$var(thermodynamics_params(ps))
+end
+# Forward parameters to CloudMicrophysics
+ρ_cloud_liq(ps::AKP) = CM.Parameters.ρ_cloud_liq(microphysics_params(ps))
```

Comment:
This function is not used. Unlike thermo parameters, microphysics parameters are not used individually in the KiD module. Maybe we don't need to forward parameters to CloudMicrophysics.